src/api/client.ts

```diff
@@ -0,0 +1,10 @@
+//공통 설정
+import axios from "axios";
+
+export const client = axios.create({
+  baseURL: import.meta.env.VITE_API_BASE_URL,
+  timeout: 5000,
```

Comment:
소소하게 큰 숫자를 사용할 때는 numeric separator를 이용해보시면 좋을거 같네요
```suggestion
  timeout: 5_000,
```

Reply:
아래처럼 표시해도 조금 더 직관적으로 얼마만큼의 시간이 timeout으로 설정되었는지 보기 좋을거 같아요
```suggestion
  timeout: 5 * 1_000,
```